docs/specs/build/config/basics.yml

```diff
@@ -162,3 +162,39 @@ outputs:
     { "content": "" }
   docs/a.mta.json: |
     { "key": "another value" }
+---
+# routes
+inputs:
+  docfx.yml: |
+    routes:
+      docs/a.md: output
+      docs/b/: output
+  docs/a.md:
+  docs/b/b.md:
+outputs:
+  output/a.json: |
+    { "content": "" }
+  output/b.json: |
+    { "content": "" }
+---
+# latter routes rule takes precedence - 1
+inputs:
+  docfx.yml: |
+    routes:
+      docs/a/: output
+      docs/a/b.md: outputB
+  docs/a/b.md:
+outputs:
+  outputB/b.json: |
+    { "content": "" }
+---
+# latter routes rule takes precedence - 2
```

Comment:
add case:
routes:
  docs/a.md: outputA
  docs/a.md: outputB

Reply:
This will fail YAML deserializer: "ArgumentException: An item with the same key has already been added. Key: docs/a.md
"

---
In reply to: [188158219](https://github.com/dotnet/docfx/pull/2760#discussion_r188158219) [](ancestors = 188158219)

Reply:
but we should support that, right?

---
In reply to: [188171713](https://github.com/dotnet/docfx/pull/2760#discussion_r188171713) [](ancestors = 188171713,188158219)

Reply:
Is there a valid user scenario for this support?

---
In reply to: [188172230](https://github.com/dotnet/docfx/pull/2760#discussion_r188172230) [](ancestors = 188172230,188171713,188158219)

Reply:
ok

---
In reply to: [188185403](https://github.com/dotnet/docfx/pull/2760#discussion_r188185403) [](ancestors = 188185403,188172230,188171713,188158219)